src/renault_api/cli/renault_vehicle.py

```diff
@@ -153,13 +154,17 @@ async def display_status(
     vehicle = await get_vehicle(websession, ctx_data)
     status_table: Dict[str, Any] = {}
 
-    await update_battery_status(vehicle, status_table)
-    await update_charge_mode(vehicle, status_table)
-    await update_cockpit(vehicle, status_table)
-    await update_location(vehicle, status_table)
-    await update_lock_status(vehicle, status_table)
-    await update_res_state(vehicle, status_table)
-    await update_hvac_status(vehicle, status_table)
+    await update_battery_status(vehicle, status_table, ctx_data))
+    await update_charge_mode(vehicle, status_table, ctx_data))
+    await update_cockpit(vehicle, status_table, ctx_data))
+    await update_location(vehicle, status_table, ctx_data))
+    await update_lock_status(vehicle, status_table, ctx_data))
+    await update_res_state(vehicle, status_table, ctx_data))
+    await update_hvac_status(vehicle, status_table, ctx_data))
```

Comment:
Remove trailing brackets here
```suggestion
    await update_battery_status(vehicle, status_table, ctx_data)
    await update_charge_mode(vehicle, status_table, ctx_data)
    await update_cockpit(vehicle, status_table, ctx_data)
    await update_location(vehicle, status_table, ctx_data)
    await update_lock_status(vehicle, status_table, ctx_data)
    await update_res_state(vehicle, status_table, ctx_data)
    await update_hvac_status(vehicle, status_table, ctx_data)
```

Reply:
copy/paste fail